src/spikeinterface/core/template.py

```diff
@@ -352,6 +352,55 @@ def to_json(self):
     def from_json(cls, json_str):
         return cls.from_dict(json.loads(json_str))
 
+    def select_units(self, unit_ids):
```

Comment:
Added this and test in https://github.com/SpikeInterface/spikeinterface/pull/2989

Reply:
@yger : could you update from main so ?